system-test/read-rows.ts

```diff
@@ -186,34 +174,46 @@ describe('Bigtable/Table', () => {
     });
 
     afterEach(() => {
-      clock.restore();
       stub.restore();
     });
 
     tests.forEach(test => {
-      it(test.name, () => {
+      it(test.name, (done: mocha.Done) => {
```

Comment:
If you want to avoid the extra mocha dependency, you can also just do `done => {` here. TS will figure it out.